graphql/studentPortal/users/fetchUserRole.ts

```diff
@@ -1,18 +1,21 @@
 import { gql } from "@apollo/client";
 
 export const FETCH_USER_ROLE = gql`
-  query FetchUserRole($_id: String = "") {
-    users(where: { id: { _eq: $_id } }) {
-      userRole {
-        value
-      }
+query FetchUserRole($_id: String = "") {
+  users(where: {id: {_eq: $_id}}) {
+    created_at
```

Comment:
this query is fine, but instead of setting this value in the React app, we should just use the default values in hasura to automatically set the created_at property when a new user is created

Reply:
That is what's done.  I'm just reading it here from the hasura database.